src/app/app.module.ts

```diff
@@ -15,9 +16,12 @@ import { NbAppComponent } from './app.component';
 import { NbLayoutDirectionToggleComponent } from './layout-direction-toggle/layout-direction-toggle.component';
 import { NbDynamicToAddComponent } from '../playground/shared/dynamic.component';
 import { NbPlaygroundSharedModule } from '../playground/shared/shared.module';
+import { NbLayoutThemeToggleComponent } from './layout-theme-toggle/layout-theme-toggle.component';
+
 @NgModule({
   imports: [
     BrowserModule,
+    BrowserAnimationsModule,
```

Comment:
should we add it here or to playground.module?

Reply:
`BrowserAnimationsModule` can be imported only in `AppModule`.
https://github.com/telerik/kendo-angular/issues/454#issuecomment-323723592